src/west/commands/common.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2019, Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''West common commands'''
+
+import argparse
+import textwrap
+
+from west.commands import WestCommand
+from west.util import WestReload
+
+class Help(WestCommand):
+    def __init__(self):
+        super().__init__(
+            'help',
+            'print west command help text',
+            _wrap('''
```

Comment:
When the help text is only a single line, you should be able to just use a single quoted string. We use triple quotes (and the related helper functions for controlling whitespace) only when there are multiple lines.

Reply:
ok!

Reply:
Done